common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java

```diff
@@ -348,7 +345,7 @@ public boolean equals(Object obj) {
      * 
      * @return
      */
-	public boolean autoActivationContentAssistantAfterApplication() {
+	public boolean isAutoActivationContentAssistantAfterApplication() {

```

Comment:
This is another dead/not used API in TextProposal. But we need this method for angularjs now. There is no need to rename it, though.